src/Microsoft.ML.ImageAnalytics/ExtensionsCatalog.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.ImageAnalytics;
+using Microsoft.ML.Transforms;
+
+namespace Microsoft.ML
+{
+    public static class ImageEstimatorsCatalog
+    {
+        /// <summary>
+        /// Converts the images to grayscale.
+        /// </summary>
+        /// <param name="catalog">The transform's catalog.</param>
+        /// <param name="columns">The name of the columns containing the image paths(first item of the tuple), and the name of the resulting output column (second item of the tuple).</param>
```

Comment:
>sulting output column (second item of the tuple).</param> [](start = 133, length = 57)

nit: What is our max line length guideline? I keep scrolling left/right to see the whole thing :)